infra.bs

```diff
@@ -648,14 +649,20 @@ Standard that should be reported and addressed.
 
 <h3 id=ecma-262-abstract-operations>ECMA-262 Abstract Operations</h3>
 
-<p>ECMA-262 uses a specification type named [=Completion Record=] to model exception-based control
-flow. Because <a href=#algorithm-control-flow>web specifications have first-class support for exception handling</a>,
-special care must be taken when interpreting the results of ECMA-262 abstract operations.
-[[!ECMA-262]]
+<p>ECMA-262 defines some algorithms as [=abstract operations=] which may be invoked from algorithms
+in other specifications. Some of these [=abstract operations=] return a type of value named
+[=Completion Record=], and ECMA262's calling conventions use this type to model control flow
+interruptions such as thrown exceptions. Because web specifications do not observe the same
+conventions (e.g. <a href=#algorithm-control-flow>exception handling is built in to the processing
+model</a>), preserving the operations' semantics necessarily involves interpreting [=Completion
```

Comment:
"interpreting" doesn't sound right to me. Maybe "handling"?

Reply:
Sure!